pywb/static/loadWabac.js

```diff
@@ -54,6 +55,7 @@ class WabacReplay
         archiveMod: "ir_",
         adblockUrl: this.adblockUrl,
         noPostToGet: true,
+        injectScripts: this.injectScripts.map(src => "../" + src),
```

Comment:
pywb supports static files from the root static directory or per-collection static directories: https://pywb.readthedocs.io/en/latest/manual/ui-guide.html#static-files. It might be worth applying that same logic to the injected scripts, in case users want e.g. Ruffle enabled but only on one collection?

Reply:
Oh and the `injectScripts.map(src => "../" + src)` on this line is because wabac.js actually adds the prefix `/static/proxy/`. It looked to me like maybe the original idea was for it to load absolute URLs from the live web, but the serviceworker just returned 404 when I tried that, but `../` worked.